src/main/kotlin/pt/up/fe/ni/website/backend/config/security/AuthConfig.kt

```diff
@@ -30,6 +30,7 @@ class AuthConfig(
 ) {
     @Bean
     fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
+        http.headers().frameOptions().disable()
```

Comment:
Why is this needed?

Reply:
I forgot to remove it :smile: , it was to make the h2 console run locally because it uses a lot of iframes, I'll remove it